pkgs/jni/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   `Iterable`.
 - Added `JObject.isA`, which checks whether a `JObject` is a instance of a
   java class.
+- Do not require JWT when building for desktop.
```

Comment:
JWT or JVM? 👀 
Not sure what jwt is in the context of jni. Maybe you can explain that?

Reply:
Good catch. It should be JAWT instead of JWT!